quickwit-backward-compat/build.rs

```diff
@@ -39,6 +61,16 @@ fn sample_split_metadata_for_regression() -> SplitMetadataAndFooterOffsets {
 }
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let index_metadata = sample_index_metadata_for_regression();
```

Comment:
the duplicated code is fine but can we just isolate the two parts (split and index) into two different functions?